.travis/release_ami5_bootstrap.yml

```diff
@@ -0,0 +1,75 @@
+---
+# --- Variables --- #
+
+local:
+  root_dir : <%= ENV['TRAVIS_BUILD_DIR'] %>
+
+# --- Release Manager Config --- #
+
+# Required: deployment targets
+targets:
+  - type              : "awss3"
+    access_key_id     : <%= ENV['AWS_DEPLOY_ACCESS_KEY_ID'] %>
+    secret_access_key : <%= ENV['AWS_DEPLOY_SECRET_ACCESS_KEY'] %>
+
+# Required: packages to be deployed
+packages:
+  - name     : "snowplow_ami5_bootstrap"
+    locations:
+    - bucket   : "snowplow-hosted-assets"
+      path     : "common/emr"
+      region   : "eu-west-1"
+    - bucket   : "snowplow-hosted-assets-us-east-1"
+      path     : "common/emr"
+      region   : "us-east-1"
+    - bucket   : "snowplow-hosted-assets-us-west-1"
+      path     : "common/emr"
+      region   : "us-west-1"
+    - bucket   : "snowplow-hosted-assets-us-west-2"
+      path     : "common/emr"
+      region   : "us-west-2"
+    - bucket   : "snowplow-hosted-assets-sa-east-1"
+      path     : "common/emr"
+      region   : "sa-east-1"
+    - bucket   : "snowplow-hosted-assets-eu-central-1"
+      path     : "common/emr"
+      region   : "eu-central-1"
+    - bucket   : "snowplow-hosted-assets-ap-southeast-1"
+      path     : "common/emr"
+      region   : "ap-southeast-1"
+    - bucket   : "snowplow-hosted-assets-ap-southeast-2"
+      path     : "common/emr"
+      region   : "ap-southeast-2"
+    - bucket   : "snowplow-hosted-assets-ap-northeast-1"
+      path     : "common/emr"
+      region   : "ap-northeast-1"
+    - bucket   : "snowplow-hosted-assets-ap-south-1"
+      path     : "common/emr"
+      region   : "ap-south-1"
+    - bucket   : "snowplow-hosted-assets-us-east-2"
+      path     : "common/emr"
+      region   : "us-east-2"
+    - bucket   : "snowplow-hosted-assets-ca-central-1"
+      path     : "common/emr"
+      region   : "ca-central-1"
+    - bucket   : "snowplow-hosted-assets-eu-west-2"
+      path     : "common/emr"
+      region   : "eu-west-2"
+    - bucket   : "snowplow-hosted-assets-ap-northeast-2"
+      path     : "common/emr"
+      region   : "ap-northeast-2"
```

Comment:
should we add eu-west-3?

Reply:
Yeah, good idea, but I think full region-support is more work than that. We at least first have to ask @jbeemster to create corresponding `snowplow-hosted-assets-eu-west-3` bucket.